include/mbgl/style/style.hpp

```diff
@@ -45,7 +47,7 @@ class Style {
     void setLight(std::unique_ptr<Light>);
 
     // Images
-    const Image* getImage(const std::string&) const;
+    const PremultipliedImage* getImage(const std::string&) const;
```

Comment:
To keep API compatibility `getImage` should return style::Image, so that client can get all associated metadata (stretches, sdf, pixel ratio, etc).

Reply:
Changed to `optional<Image>` in the latest commit